Orange-OpenSource/Orange-Boosted-Bootstrap
# Prepare 5.0.0-beta3

- Bootstrap v5.0.0-beta3 release: https://github.com/twbs/bootstrap/releases/tag/v5.0.0-beta3
- Bootstrap blog post: https://blog.getbootstrap.com/2021/03/23/bootstrap-5-beta-3/

---

Preview : https://deploy-preview-650--boosted.netlify.app/

---

  - Firefox ESR
  - Latest Edge, Chrome, Firefox, Safari
  - iOS Safari
  - Chrome & Firefox on Android
  - ~~if bumping a minor or major version~~ → still v5.0
  - install [Conventionnal Changelog](https://github.com/conventional-changelog/conventional-changelog) and `conventional-changelog-cli` globally
  - run `conventional-changelog -p angular -i CHANGELOG.md -s`
  - Ship list: #585
- [ ] merge on `v5-dev`
  - attach zip file
  - paste CHANGELOG or Ship List in the release's description
- [ ] We don't publish v5.0 docs yet
  - if you are already logged in NPM (with a personnal account, for example), [you'd better use a repository scoped `.npmrc` file](https://stackoverflow.com/questions/30114166/how-to-have-multiple-npm-users-set-up-locally)
  - if you're releasing a pre-release, use `--tag`, eg for v5-alpha1 `npm publish boosted-5.0.0-alpha1.tgz --tag next`
- [ ] check release on [NPM](https://www.npmjs.com/package/boosted), [Packagist](https://packagist.org/packages/orange-opensource/orange-boosted-bootstrap)…
